worker/appm/conversion/configgroup.go

```diff
@@ -71,7 +68,24 @@ func (c *configGroup) secretForConfigGroup() (*corev1.Secret, error) {
 			Namespace: c.namespace,
 			Labels:    labels,
 		},
-		Data: data,
+		Data: c.parseVariable(items),
 		Type: corev1.SecretTypeOpaque,
 	}, nil
 }
+
+func (c *configGroup) parseVariable(items []*model.ConfigGroupItem) map[string][]byte {
```

Comment:
Parsing of environment variables for all components should be supported. Instead of configuring group variables.

Reply:
fixed